content/docs/features.md

```diff
@@ -36,29 +31,35 @@ toc: false
   |5   |Do something    |Edge        |
   |6   |Verify something|Vertex      |
   |:   |:               |:           |
-  {: style="color:gray; font-size: 90%"}
-
-
-* An edge represents an action, a transition.<br>{: style=""}
 
-  An action could be an API call, a button clicked, a timeout. Anything that moves your System Under Test into a new state that you want to verify. But remember, there is no verification going on in the edge. That happens only in the vertex. 
-  {: style="color:gray; font-size: 90%"}
+* An edge represents an action or a transition.  
 
-* A vertex represents verification(s).<br>{: style=""}
+  An action could be an API call, a button clicked, a timeout etc. 
+  Anything that moves your System Under Test into a new state that you want to verify. 
+  But remember, there is no verification going on in the edge. That happens only in the vertex. 
 
-  A verification is where you would have assertions in your code. It's here where you verify that an API call returns the correct values. That a button click actually did close a dialog. Or that when the timeout should have occurred, the System Under Test triggered the expected event.
-  {: style="color:gray; font-size: 90%"}
+* A vertex represents a verification.  
 
+  A verification is where you would have assertions in your code. 
+  It's where you verify that an API call returns the correct values. 
+  That a button click actually did close a dialog. 
+  Or that when the timeout should have occurred, the System Under Test triggered the expected event.
 
-### Offline
+### Offline generation
 
 <a download="example_1.graphml" href="/content/resources/example_1.graphml"><img src="/images/example_1.png" alt="Model" align="right"/></a>
 
-The path generation is done once. It's not directly connected to any test automation code. The path needs to be stored in some intermediate format. Typically, the path is generated from command line, and the output stored on file. The content of the file is then used by your test automation to drive test.
+The path generation is done once. 
+The path needs to be stored in some intermediate format. 
+Typically, the path is generated from [command line](/cli-offline/), and the output stored on file. 
+The content of the file is then would be used by you for manual test execution or in test automation to drive a test.
 
 **Example**
 
-Given the following graph [example_1.graphml](/content/resources/example_1.graphml), a path can be generated by running a command from a terminal. Below is an example of a generated path from that graph using a random path generator with an edge coverage stop condition. The test will start at the element `app_closed`
+Given the following graph [example_1.graphml](/content/resources/example_1.graphml), 
```

Comment:
I have transferred some sentences on new lines due to some lines are very long and it is not readable in IDE.
It requires horizontal scrolling and anybody who read it may lost focus on which line he was.